exporter/clickhousetracesexporter/migrations/000014_top_level_operations_mv.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.top_level_operations (
```

Comment:
@srikanthccv I don't usage of these tables in [query-service PR](https://github.com/SigNoz/signoz/pull/1455/files).
You are still using indexTable to query top level operations.

Reply:
Yes, I should be using this `top_level_operations` in that PR.